DOCUMENTATION.md

```diff
@@ -273,6 +273,8 @@ This is an invitation-only feature. [Contact the GOV.UK Notify team](https://www
 
 To send a file by email, add a placeholder field to the template then upload a file. The placeholder field will contain a secure link to download the file.
 
+The links are unique and unguessable. GOV.UK Notify cannot access or decrypt your file without your permission.
```

Comment:
Looks good. Just one question - is there a specific way for users to give permission to GOV.UK Notify?

Reply:
We can’t decrypt service team’s files even with their permission. It’s part of the privacy built into the design of document download. Only the user who has been sent the tokenised URL can decrypt the file.

Reply:
👍 I'll remove this from all client docs.